lib/screens/gh_repo.dart

```diff
@@ -51,6 +51,14 @@ class GhRepoScreen extends StatelessWidget {
     }
   }
 
+  Future<String> _fetchContributors(BuildContext context) async {
+    final auth = Provider.of<AuthModel>(context);
+    final res = await auth.ghClient.getJSON( 
+      '/repos/$owner/$name/contributors',
+    );
+      return res.length.toString();
+  }
```

Comment:
Is `res.length` the total count of contributors, or the count of the first page?

If we could not get the exact count, perhaps it is better to leave it blank.

Reply:
Using `stats/contributors` we can get the exact count as it does not support pagination. So I think we can use this in `gh_repo` and leave `gh_contrubutors` (with `contributors`) unchanged. Is this fine? I'll make the relevant changes.

Reply:
Yeah, I think it is OK if we could get the exact count.